### PR TITLE
998 course tests (connects #998)

### PR DESCRIPTION
**Pull request by razu9861**

### CODE REVIEW

**Review by nirojdyola — COMMENTED** [review]
Do we need console.log(coursedetail) code in line number 240 (takeTestView.js) ?

**Review by nirojdyola — APPROVED** [review]

File: app/MyApp/app/views/takeTestView.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ $(function() {
         totalMarks: 0,
         preresult: null,
         attributes: {'method': 'post'},
-        mymodel: null,
+        myModel: null,
```
Comment (lmmrssa) [inline app/MyApp/app/views/takeTestView.js]: why is the name case changed?

File: app/MyApp/index.html [hunk 2]
```diff
@@ -1526,7 +1526,7 @@ <h3 class="bigheader">
                     </tr>
                     <tr>
                         <td>
-                            <input type="text" value="<%=answer%>" name="singleLineAnswer" style="margin-bottom: -45px;">
+                            <input type="text" value="<%=answer%>" name="singleLineAnswer">
```
Comment (lmmrssa) [inline app/MyApp/index.html:1529]: these style has been removed. are they added somewhere else or are they not required at all?

Comment (Pr0chin) [reply app/MyApp/index.html:1529]: Removing margin for input field attributes

**Review by Pr0chin — APPROVED** [review]

File: app/MyApp/index.html (hunk 2 above)
Comment (razu9861) [reply app/MyApp/index.html:1529]: they are not required at all

**Review by lmmrssa — APPROVED** [review]

**Review by Rupesh87 — APPROVED** [review]

**Pull request closed without merging**